scripts/launch-xrd

```diff
@@ -244,10 +245,6 @@ while [[ $# -gt 0 ]]; do
             ;;
         -- )
             shift
-            if [[ ${#CTR_MGR_ARGS[@]} -ne 0 ]]; then
-                echo "When using --, use container manager args after it. These args are inavlid for the script ${CTR_MGR_ARGS[*]}" >&2
-                bad_usage
-            fi
             CTR_MGR_ARGS+=("$@")
             shift $#
```

Comment:
Logically the loop is over, so `break` rather than shifting by the number of the rest of the args.